tests/manifests/stability/stability.yaml

```diff
@@ -45,18 +45,11 @@ spec:
     image: ""
     imagePullPolicy: Always
     command:
-    - /bin/sh
-    - -c
-    - |
-      set -e
-      for i in `seq 1 100`
-      do
-        echo "######################## $i begin ########################"
-        /usr/local/bin/stability-test \
-        --config=/etc/tidb-operator-stability/config.yaml \
-        --slack-webhook-url=""
-        echo "######################## $i end ##########################"
-      done
+    - /usr/local/bin/stability-test
+    - --config=/etc/tidb-operator-stability/config.yaml
+    - --operator-image=pingcap/tidb-operator:v1.0.0-beta.2
+    - --operator-tag=v1.0.0-beta.2
```

Comment:
You can define `image-repo` and `image-tag` to avoid passing the same version twice.

Reply:
the `operator-tag` is used by git clone to check out the source code.

For example, the image maybe `pingcap/tidb-operator:latest` and the `operator-tag ` is `master`.